asserts/database.go

```diff
@@ -574,6 +586,78 @@ func (db *Database) FindManyPredefined(assertionType *AssertionType, headers map
 	return db.findMany([]Backstore{db.trusted, db.predefined}, assertionType, headers)
 }
 
+// FindSequence finds an assertion for the given headers and after for
+// a sequence-forming type.
+// The provided headers must contain a sequence key, i.e. a prefix of
+// the primary key for the assertion type except for the sequence
+// number header.
+// The assertion is the first in the sequence under the sequence key
+// with sequence number > after.
+// If after is -1 it returns instead the assertion with the largest
+// sequence number.
+// It will constraint itself to assertions with format <= maxFormat
+// unless maxFormat is -1.
+// It returns a NotFoundError if the assertion cannot be found.
+func (db *Database) FindSequence(assertType *AssertionType, sequenceHeaders map[string]string, after, maxFormat int) (SequenceMember, error) {
+	err := checkAssertType(assertType)
+	if err != nil {
+		return nil, err
+	}
+	if !assertType.SequenceForming() {
+		return nil, fmt.Errorf("cannot use FindSequence with not sequence-forming assertion type %q", assertType.Name)
```

Comment:
s/not/non/ ? (not sure, cc @degville )

Reply:
> s/not/non/ ? (not sure, cc @degville )

you're right - it should be `cannot use FindSequence with non sequence-forming assertion type`